src/site/_includes/js/test/eligibility-programs.test.js

```diff
@@ -2146,4 +2147,31 @@ describe('Program eligibility', () => {
         .is([elig.cnst.wic.CHILD_EXIT_AGE - 1, 99]);
     });
   });
+  describe('Clipper START Program', () => { // Broken!
+    test('Not eligible with default input', () => {
+      expect(elig.clipperStartResult(input).eligible).not.toBe(true);
+    });
+
+    test('Cannot have RTC Clipper Card', () => {
+      input.existingRtcClipperMe = false;
+      input.income.valid = true;
+      input.age = elig.cnst.clipper.MIN_ELIGIBLE_AGE;
+      check(elig.clipperStartResult, input).isNotEligibleIf('existingRtcClipperMe').is(true);
+    });
+
+    test('Age must be from 19 to 64 years', () => {
```

Comment:
I suggest you change this to `'Age must be within the limits'` so that there is no risk of the test description getting out of date with the actual code & logic.

Reply:
Done in [f3038e7](https://github.com/theunitedeffort/theunitedeffort.org/pull/610/commits/f3038e71984f7e2cbbb53b7d59b5ede13d9315f8)